GraphicEditor/View/UserControls/HospitalMapUserControl.xaml

```diff
@@ -3,29 +3,204 @@
+                            Width="210"
+                            Height="123"
+                            Cursor="Hand" Margin="288,63,683,462"
+                            Command="{Binding Path=NavCommand}"
+                            CommandParameter="emergency">
+                    <StackPanel>
+                        <TextBlock Style="{StaticResource MaterialDesignHeadline5TextBlock}" 
+                                       TextWrapping="Wrap">
+                                Emergency Ambulance
+                        </TextBlock>
+                        <Image Source="/GraphicEditor;component/Resources/Images/hospitalLogo.png" 
+                                   Height="30" Margin="0,3,0,0"/>
+                    </StackPanel>
+                </Button>
+
+                <!--Cardiology button-->
+                <Button 
+                        HorizontalAlignment="Center"
+                        VerticalAlignment="Center"
+                        Background="Orange" 
+                        BorderBrush="Black"
+                        BorderThickness="3"
+                        Width="188"
+                        Height="122"
+                        Cursor="Hand" Margin="509,64,484,462"
+                        Command="{Binding Path=NavCommand}"
+                        CommandParameter="cardiology">
+                    <StackPanel>
+                        <TextBlock Style="{StaticResource MaterialDesignHeadline5TextBlock}" 
+                                       TextWrapping="Wrap">
+                                Cardiology
+                        </TextBlock>
+                        <Image Source="/GraphicEditor;component/Resources/Images/hospitalLogo.png" 
+                                   Height="30" Margin="0,3,0,0"/>
+                    </StackPanel>
+                </Button>
+
+                <!--Orthopedy button-->
+                <Button 
+                        HorizontalAlignment="Center"
+                        VerticalAlignment="Center"
+                        Background="OrangeRed" 
+                        BorderBrush="Black"
+                        BorderThickness="3"
+                        Width="146"
+                        Height="205"
+                        Cursor="Hand" 
+                        Margin="517,393,518,50"
+                        Command="{Binding Path=NavCommand}"
+                        CommandParameter="orthopedy">
+                    <StackPanel>
+                        <TextBlock Style="{StaticResource MaterialDesignHeadline6TextBlock}" 
+                                   TextWrapping="Wrap">
+                                Orthopedy
+                        </TextBlock>
+                        <Image Source="/GraphicEditor;component/Resources/Images/hospitalLogo.png" 
+                                   Height="30" Margin="0,3,0,0"/>
+                    </StackPanel>
+                </Button>
+
+                <!--Pediatry button-->
```

Comment:
The word you're looking for is Pediatrics 😄 if you're not sure just look up the term in Serbian on Wikipedia and then change the language to English so you'll get the proper term.